### PR TITLE
chore: upgrade typeorm

### PR DESCRIPTION
**Pull request by ahmadnazir**
# Description

> Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Upgrading typeorm to `0.2.35` which comes with a few bug fixes: https://github.com/typeorm/typeorm/releases/tag/0.2.35

# Type of change

> Please select the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (non-breaking change which does not add functionality)

# Checklist:

- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: lib/common/RLSPostgresQueryRunner.ts [hunk 1]
```diff
@@ -27,14 +27,18 @@ export class RLSPostgresQueryRunner extends PostgresQueryRunner {
     this.actorId = tenancyModelOptions.actorId;
   }
 
-  async query(queryString: string, params?: any[]): Promise<any> {
+  async query(
+    queryString: string,
+    params?: any[],
+    useStructuredResult?: boolean,
```
Comment (stelescuraul) [inline lib/common/RLSPostgresQueryRunner.ts:33]: This was the missing property that triggered the error ?

**Review by stelescuraul — CHANGES_REQUESTED** [review]
Let's add a few tests to make sure that the relation loader is working fine with the RLS. 
Few tests that should cover selecting relations, with/without conditions as well

File: lib/common/RLSPostgresQueryRunner.ts (hunk 1 above)
Comment (ahmadnazir) [reply lib/common/RLSPostgresQueryRunner.ts:33]: Yes.

**Comment by ahmadnazir** [comment]
Closing in favor of https://github.com/Avallone-io/rls/pull/50

**Pull request closed without merging**